alerts/cluster-monitoring-operator/PrometheusDuplicateTimestamps.md

```diff
@@ -40,13 +40,14 @@ Consequently, queries may yield incorrect and unexpected results.
 ## Mitigation
 
 If the alert originates from the `openshift-monitoring` namespace, please open a
-support case. If not, proceed based on the reviewed logs for the affected
-Prometheus instance:
+support case. If not, this might be due to one or both of the following issues:
 
-### The logs reveal one of the following issues
+### Target duplication
 
-- The same target is defined in different scrape pools
-- Distinct targets across different scrape pools are producing the same samples
+The logs might help determine the following information:
```

Comment:
```suggestion
The logs could help differentiating between the following situations:
```

Reply:
Actually, one may have both issues at the same time, and I'd like both cases to be considered (not to choose one of them)